Assets/Game/Scripts/InspectorField/TagDropDown/TagSelectorPropertyDrawer.cs

```diff
@@ -0,0 +1,76 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.Collections.Generic;
+ 
+//Original by DYLAN ENGELMAN http://jupiterlighthousestudio.com/custom-inspectors-unity/
+//Altered by Brecht Lecluyse http://www.brechtos.com
+ 
+[CustomPropertyDrawer(typeof(TagSelectorAttribute))]
+public class TagSelectorPropertyDrawer : PropertyDrawer
+{
+ 
+    public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+    {
+        if (property.propertyType == SerializedPropertyType.String)
+        {
+            EditorGUI.BeginProperty(position, label, property);
+ 
+            var attrib = this.attribute as TagSelectorAttribute;
+ 
+            if (attrib.UseDefaultTagFieldDrawer)
+            {
+                property.stringValue = EditorGUI.TagField(position, label, property.stringValue);
+            }
+            else
+            {
+                //generate the taglist + custom tags
+                List<string> tagList = new List<string>();
+                tagList.Add("<NoTag>");
```

Comment:
What GameObject will use the NoTag? And how it is used?

Reply:
I copy this code somewhere. I think `<NoTag>` just means empty string, we can modify it if need to.